redbox-core/redbox/models/file.py

```diff
@@ -13,6 +13,18 @@
 
 
 class ProcessingStatusEnum(str, Enum):
+    """Current status of the file processing.
+
+    Note: The Django app interprets these as:
+    "processing" -> "processing"
+    "complete" -> "complete"
+    "failed" -> "errored"
+    anything else -> "processing"
+
+    If you add any other fail state options, the Django app will need to be updated.
+    django_app/redbox_app/redbox_core/models.py File.update_status_from_core
+    """
```

Comment:
Is there a more elegant approach here than just pointing out the risk?

Reply:
I dont think so. not without boiling the ocean anyway :)